### ISSUE TITLE
could not determine kind of name for C.uint32_t

### ISSUE THREAD

**Issue by neclepsio** [mkb218/gosndfile#13]
I successfully compiled under Windows and MSYS2, but under Ubuntu I get this error: "could not determine kind of name for C.uint32_t".

For both I used Go 1.8.

Go env is:
```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/ignazio/Programming/Go-Workspace"
GORACE=""
GOROOT="/home/ignazio/bin/go"
GOTOOLDIR="/home/ignazio/bin/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build419818202=/tmp/go-build -gno-record-gcc-switches"
CXX="g++"
CGO_ENABLED="1"
PKG_CONFIG="pkg-config"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
```

**Comment by mkb218** [comment]
Thanks. Are you building with gccgo? I haven't tested with this.

**Comment by neclepsio** [comment]
No, I'm just building with plain `go build` and just set `GOROOT` and `GOPATH`. I'm using `libsndfile1-dev` package.

**Comment by mkb218** [comment]
Got it.

**Comment by neclepsio** [comment]
I found out: Ubuntu ships libsoundfile 1.0.25 while MSYS2 1.0.26. Between the two, `SF_BROADCAST_INFO_VAR.time_reference_low` and `SF_BROADCAST_INFO_VAR.time_reference_high` have different types: `uint32_t` for libsoundfile 1.0.26 shipped with MSYS2, `unsigned int` for version 1.0.25 shipped with Ubuntu. 

Maybe you should document that not only tests fail, but also cannot compile with versions less than 1.0.26. Or maybe ship an alternative build tag with modified `BroadcastInfo` struct.

**Comment by mkb218** [comment]
Thanks for digging into this.
